src/panel/MasqStatus.js

```diff
@@ -0,0 +1,89 @@
+/* globals _ */
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import nconf from '@qwant/nconf-getter';
+import MasqOnboardingModal from '../modals/masq_onboarding_modal';
+import Telemetry from '../libs/telemetry';
+
+const masqOnboardingModal = new MasqOnboardingModal();
+
+export default class MasqStatus extends Component {
+  static propTypes = {
+    store: PropTypes.object.isRequired,
+    user: PropTypes.shape({
+      username: PropTypes.string.isRequired,
+      profileImage: PropTypes.string,
+      defaultProfileImage: PropTypes.object.isRequired,
+    }),
+  }
+
+  login = async () => {
+    try {
+      Telemetry.add(Telemetry.MASQ_MENU_ACTIVATE);
+      await this.props.store.login();
+    } catch (e) {
+      console.warn(`An exception occurred in MasqStatus::login: ${e}`);
+    }
+  }
+
+  logout = async () => {
+    Telemetry.add(Telemetry.MASQ_MENU_DESACTIVATE);
+    await this.props.store.logout();
+  }
+
+  openMasqOnboarding = () => {
+    Telemetry.add(Telemetry.MASQ_MENU_ONBOARDING);
+    masqOnboardingModal.open();
+  }
+
+  openMasq = () => {
+    Telemetry.add(Telemetry.MASQ_MENU_OPEN);
+    window.open(nconf.get().masq.baseMasqAppUrl, '_blank');
+  }
+
+  render() {
+    const user = this.props.user;
+    const isLoggedIn = !!user;
+
+    return <div className="masqStatus">
+      <div className="icon-masq masqStatus__masq_icon" onClick={this.openMasq} />
+      {isLoggedIn
+        ? <Fragment>
```

Comment:
we can achieve better readability if we split these two components into two render functions like `renderEnabled` and `renderDisabled` for instance, what do you think ?

Reply:
and Fragment will not be required this way

Reply:
I'm accustomed to reading stuff like that but it's true that it may be confusing, and there is a clear case here :) I'll change, thanks.

Reply:
Done :)